databricks/koalas/tests/test_dataframe.py

```diff
@@ -489,3 +488,21 @@ def test_sample(self):
             kdf.sample()
         with self.assertRaises(NotImplementedError):
             kdf.sample(n=1)
+
+    def test_concat(self):
```

Comment:
we should create a new file for the general utility functions defined in namespace.py

Reply:
okie